crates/eframe/src/native/run.rs

```diff
@@ -629,8 +629,8 @@ mod glow_integration {
             Ok(())
         }
 
-        fn window(&self) -> &winit::window::Window {
-            self.window.as_ref().expect("winit window doesn't exist")
+        fn try_window(&self) -> Option<&winit::window::Window> {
```

Comment:
No need for the extra `try_` prefix methinks

```suggestion
        fn window(&self) -> Option<&winit::window::Window> {
```

Reply:
Fixed. Also removed the unnecessary error log as it's created for the first expect and checked for none before the second expect.